infra/gcp/ensure-prod-storage.sh

```diff
@@ -220,4 +220,9 @@ empower_group_to_admin_artifact_auditor \
 empower_artifact_auditor "${PROD_PROJECT}"
 empower_artifact_auditor_invoker "${PROD_PROJECT}"
 
+# Special case: empower Prow account to have access to GCR for Workload Identity.
+empower_prowacct_for_workload_identity \
```

Comment:
IMO this should be more like:
* `empower_pods_to_identify_as_gcp_service_account`
* `bind_k8s_service_account_to_act_as_gcp_service_account`